access-control-app/int-test/cmr/access_control/int_test/permission_check_test.clj

```diff
@@ -920,6 +921,59 @@
             "user1" ["update" "delete"]
             "user2" []))))))
 
+(defn get-permissions-2
+  "Helper to get permissions with the current context and the specified username string or user type keyword and concept ids."
+  [user & concept-ids]
+  (json/parse-string
+    (ac/get-permissions
```

Comment:
Non macro indentation is 1

Reply:
Oops, that function ended up being not needed since I accomplished what I needed without changing that function. Deleted now.